project/Jdk9.scala

```diff
@@ -11,36 +11,87 @@
  * Copyright (C) 2017-2022 Lightbend Inc. <https://www.lightbend.com>
  */
 
-import sbt._
 import sbt.Keys._
+import sbt._
 
 object Jdk9 extends AutoPlugin {
```

Comment:
shouldn't this be renamed - it has nothing to do with Jdk9 now

Reply:
I think we can just leave it for now, I changed it to `MultiJavaLTS` support, and then there are many place need to update.

Reply:
If you like, I think I can create a pending PR to do this, but as I say, many places need to be updated.